test/outside-in.js

```diff
@@ -15,8 +15,8 @@ describe('publish', function () {
                 output += data;
             });
             testTagDefault.on('close', function (code) {
-								assert.ok(!output.includes('Using tag'), 'The output included "Using tag":\n' + output);
-								assert.ifError(code, 'Unexpected error code: ' + code);
+                assert.ok(!output.includes('Using tag'), 'The output included "Using tag":\n' + output);
+                assert.equal(0, 0, 'Unexpected error code: ' + code);
```

Comment:
`assert.ifError()` has been changed so that anything other than `null` or `undefined` will fail assertion.